### ISSUE TITLE
Saggital Slices

### ISSUE THREAD

**Issue by alex-attinger** [cortex-lab/allenCCF#28]
First of all, thanks a lot for making this tool available! I was wondering if there is a way to make it work with sagittal slices?

**Comment by philshams** [comment]
I've just added this functionality (and transverse slices too).

In _Navigate_Atals_and_Register_Slices.m_, simply add your desired plane
```
% plane to view ('coronal', 'sagittal', 'transverse')
plane = 'coronal';
```
This will apply the relevant permutation to the 3D brain array. You'll also now need to put down this same parameter in the _Display_Probe_Track.m_ script, in the _additional parameters_ section.

**Comment by kouichi-c-nakamura** [comment]
For clarity, does [this commit mentioned above](https://github.com/cortex-lab/allenCCF/commit/427d64c3c0772f7be4a138ed55d2ae18db626586) allow registering sagittal sections? Not just browsing?

**Comment by philshams** [comment]
Yes, the procedure for transformation/registering slices should work the same as with coronal sections. The only difference is that the clicked points' x, y, and z axes will be flipped around. So you'll need to include the _plane_ parameter in the Display_Probe_Track.m script (or change the axes accordingly anywhere else you use those points).

**Comment by alex-attinger** [comment]
Thank you! The atlas viewer now displays sagittal views of the brain. However, when I try to register and transform slices from the slice viewer, the slices are distorted (see below). I am assuming I need to flip some of the coordinates as well?
![image](https://user-images.githubusercontent.com/2227711/66870650-c3a35d00-ef56-11e9-9cd2-94296ebb160e.png)

**Comment by alex-attinger** [comment]
Thank you! Atlas Viewer now displays sagittal slices. However, there seems to be something off with the transform (see below). Any ideas how it could be fixed?
![image](https://user-images.githubusercontent.com/2227711/66871399-6d371e00-ef58-11e9-869f-166db7532f0c.png)

**Comment by philshams** [comment]
Thank you -- I've just pushed that change

**Comment by juliencarponcy** [comment]
Hello, 

Just a comment to say that browsing through sagittal sections there is a minor bug that I've spotted so far. In the atlas viewer, you have to switch to mode "scroll along A/P axis" to actually go over different slices along the Medio-Lateral axis to browse between different para-sagittal plans. An the scroll along slice images does not do anything besides changing the name of Slice 2. 3 . 4.... in the top left corner of the Atlas viewer. I still have to use it more to see whether that could affects results or if it is just a display issue. 

Thanks for your work,

Julien

**Comment by kouichi-c-nakamura** [comment]
Made a Pull Request 
https://github.com/cortex-lab/allenCCF/pull/58

**Comment by kouichi-c-nakamura** [comment]
Here is proof that the tool really works for parasagittal sections (BTW, I think it's officially called "parasagittal", rather than "sagittal," meaning that it is parallel to the sagittal plane which lies at the midline)

![MicrosoftTeams-image](https://user-images.githubusercontent.com/6400352/180310743-6a4c903a-9316-47a8-bd4a-8070828a167c.png)
